web/webpack.config.js

```diff
@@ -95,17 +96,20 @@ module.exports = {
       // TODO: modify it to not depend on cockpit
       // forward the manifests.js request and patch the response with the
       // current Agama manifest from the ./src/manifest.json file
-      "/manifests.js": {
-        target: cockpitTarget + "/cockpit/@localhost/",
-        // ignore SSL problems (self-signed certificate)
+      // "/manifests.js": {
+      //   target: cockpitTarget + "/cockpit/@localhost/",
```

Comment:
Question: why keep comments with Cockpit references. Nothing against Cockpit, of course. I just think these lines are no longer useful.

Reply:
It is not because of cockpit, but because of the `manifest.js`. We can remove them when we handle the manifest properly.